microsoftgraph/microsoft-graph-explorer-v4
# Task: Make resources tab consistent with kiota vs code openapi explorer

## Overview
Closes #2641 

### Demo

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
Click on the Resources tab
Notice the changes in icons and links
